extensions/notebook/resources/notebooks/JupyterBooksCreate.ipynb

```diff
@@ -112,54 +130,108 @@
                 "firstLevelUrls = re.findall(r'^(?:\\s+$[\\r\\n]+)+(\\- url: [a-zA-Z0-9\\\\.\\s\\-\\/]+$[\\r\\n]+)', result, re.MULTILINE)\n",
                 "urls = re.findall(r'- url: [a-zA-Z0-9\\\\.\\s\\-\\/]+$', result, re.MULTILINE)\n",
                 "headers = re.findall(r'- header: [a-zA-Z0-9\\\\.\\s-]+$', result, re.MULTILINE)\n",
+                "# all the markdown urls are placed at the end of the list. \n",
+                "possibleMarkdowns = re.findall(r'(- url: [a-zA-Z0-9\\\\.\\s\\-\\/]+$[\\r\\n]+)[\\r\\n]', result, re.MULTILINE)\n",
+                "\n",
+                "def getMarkdownFile(url):\n",
+                "    # url are usually defined in toc as: \"- url: <Path\\to\\notebookFile>\"\n",
+                "    # substring from 7th postion excluding the \"- url: \" from the path\n",
+                "    #folders = url[7:].rstrip().split(path.sep)\n",
+                "    if (os.name == 'nt'):\n",
+                "        # if windows get the drive letter and add it to the url\n",
+                "        driveletter = content_folder.split(path.sep)[0]\n",
```

Comment:
Do we support UNC paths for this?

Reply:
no we don't, but this is one more issue that I've seen with jupyter formatting the toc file. It removed the drive letter and specifies the urls.